visualization.shadertoy/platforms.txt

```diff
@@ -0,0 +1 @@
+all
```

Comment:
well, that was meant as all supported platform. so please:
linux osx windows rbpi